backend/src/routes/user.js

```diff
@@ -67,6 +68,7 @@ router.delete(
   async function (req, res, next) {
     try {
       const { id } = req.params;
+      await postModel.deleteMany({ post_person_id: id });
```

Comment:
This is in try block, What if a user has not created any of the post? will it catch an error? could you test it in that way

Reply:
Confirmed even if the user has not created any post it will not throw an error and it should work just fine.